crates/core/src/backend/decrypt.rs

```diff
@@ -31,6 +31,8 @@ pub trait DecryptFullBackend: DecryptWriteBackend + DecryptReadBackend {}
 
 impl<T: DecryptWriteBackend + DecryptReadBackend> DecryptFullBackend for T {}
 
+pub type StreamResult<Id, F> = RusticResult<Receiver<RusticResult<(Id, F)>>>;
```

Comment:
Do we need to export this for public API?

Reply:
no, is changed.